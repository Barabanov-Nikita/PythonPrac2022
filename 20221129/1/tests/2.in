class C(metaclass=dump):
    def __init__(self, *args, **kwargs):
        self.args = args
        self.kwargs = kwargs

    def add(self, *args, **kwargs):
        self.args = self.args + args
        self.kwargs.update(kwargs)
        return self.args, self.kwargs


c = C(10, k=5)
print(c.add(9, w=6))
print(c.add(4, a="qq"))