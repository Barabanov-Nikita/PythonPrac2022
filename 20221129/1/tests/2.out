__init__: (10,), {'k': 5}
add: (9,), {'w': 6}
((10, 9), {'k': 5, 'w': 6})
add: (4,), {'a': 'qq'}
((10, 9, 4), {'k': 5, 'w': 6, 'a': 'qq'})
